[2022-02-03 14:49:09,410] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 14:49:09,421] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 14:49:09,421] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:49:09,422] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 14:49:09,422] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:49:09,432] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-01 00:00:00+00:00
[2022-02-03 14:49:09,436] {standard_task_runner.py:52} INFO - Started process 1495 to run task
[2022-02-03 14:49:09,441] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-12-01T00:00:00+00:00', '--job-id', '464', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpo4dpivtn', '--error-file', '/tmp/tmpogyqw9uu']
[2022-02-03 14:49:09,442] {standard_task_runner.py:77} INFO - Job 464: Subtask local_to_gcs_task
[2022-02-03 14:49:09,495] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 14:49:09,530] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 14:49:09,551] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-01T00:00:00+00:00
[2022-02-03 14:49:11,522] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 14:49:11,541] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20191201T000000, start_date=20220203T144909, end_date=20220203T144911
[2022-02-03 14:49:11,571] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 14:49:11,602] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-03 15:13:46,656] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 15:13:46,670] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 15:13:46,670] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:13:46,671] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 15:13:46,671] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:13:46,682] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-01 00:00:00+00:00
[2022-02-03 15:13:46,694] {standard_task_runner.py:52} INFO - Started process 3081 to run task
[2022-02-03 15:13:46,702] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-12-01T00:00:00+00:00', '--job-id', '566', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpswx03szh', '--error-file', '/tmp/tmp6exbc4x1']
[2022-02-03 15:13:46,705] {standard_task_runner.py:77} INFO - Job 566: Subtask local_to_gcs_task
[2022-02-03 15:13:46,786] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-12-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 15:13:46,830] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 15:13:46,852] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-01T00:00:00+00:00
[2022-02-03 15:13:49,335] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 15:13:49,350] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20191201T000000, start_date=20220203T151346, end_date=20220203T151349
[2022-02-03 15:13:49,414] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 15:13:49,445] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
